### ISSUE TITLE
Resetting stub mappings with mappings from classpath resources is broken in 2.4.1

### ISSUE THREAD

**Issue by otrosien** [tomakehurst/wiremock#536]
Probably caused by the change in 44c25e422f9970cc792c385a908fa07348a75488 it is no longer possible to register stubMappings via classpath resources and later call `resetMappings()`. 

### Scenario

I publish wiremock stubs in artifactory and want to be able to load them in my downstream services for integration testing. Between the tests I need to be abe to reset the stub mappings.

Observed behaviour: resetMappings() causes an UnsupportedOperationException:
```
java.lang.UnsupportedOperationException: Classpath file sources are read-only
	at com.github.tomakehurst.wiremock.common.ClasspathFileSource.deleteFile(ClasspathFileSource.java:198) ~[wiremock-2.4.1.jar:na]
	at com.github.tomakehurst.wiremock.standalone.JsonFileMappingsSource.removeAll(JsonFileMappingsSource.java:72) ~[wiremock-2.4.1.jar:na]
	at com.github.tomakehurst.wiremock.core.WireMockApp.resetMappings(WireMockApp.java:243) ~[wiremock-2.4.1.jar:na]
	at com.github.tomakehurst.wiremock.WireMockServer.resetMappings(WireMockServer.java:336) ~[wiremock-2.4.1.jar:na]
	at com.epages.wiremock.starter.WireMockListener.beforeTestMethod(WireMockListener.java:74) ~[wiremock-spring-boot-starter-0.7.16-SNAPSHOT.jar:na]
```

### Desired behaviour

The mappings should be unregistered, but of course there should be no file deletions.

### Steps to reproduce

I have just tried to upgrade my project to 2.4.1 (https://github.com/ePages-de/restdocs-wiremock/issues/40), but if you like I can setup a minimal test case to demonstrate this issue.

**Comment by tomakehurst** [comment]
Thanks for pointing this out. Will fix in the next patch release.

**Comment by tomakehurst** [comment]
Fixed by c7597a766aaad79d895deae834145b3cb6b5753a

**Comment by otrosien** [comment]
Thanks!
